skyportal/tests/api/test_candidates.py

```diff
@@ -216,7 +216,9 @@ def test_token_user_cannot_post_two_candidates_same_obj_filter_passed_at(
         },
         token=upload_data_token,
     )
-    assert status in [401, 500]
```

Comment:
Its quite amazing that we were actually testing for this and excepting the wrong return code. Good catch!

Reply:
There that mystery is solved :)